### DISCUSSION TITLE
using cc2531  as extender

### DISCUSSION THREAD

**Discussion by pol-henri** [Koenkk/zigbee2mqtt#7147] [Devices]
Hello, I would have liked to know if it was possible to use a cc2531 key as an "extender", I understood that it was possible to extend the range of the network via this one, where well the powered devices are the only ones able to extend the quality of the network? thank you for your clarification

**Comment by Koenkk** [comment]
https://www.zigbee2mqtt.io/information/supported_adapters.html#router

**Reply by pol-henri** [reply]
Thanks, I begin with zigbee network and clearly I need to read more about thanks
